src/EnergyPlus/DaylightingManager.cc

```diff
@@ -1617,9 +1572,7 @@ void FigureDayltgCoeffsAtPointsSetupForWindow(
     Vector3<Real64> &VIEWVC2,                 // Virtual view vector in absolute coordinate system
     bool &is_Rectangle,                       // True if window is rectangular
     bool &is_Triangle,                        // True if window is triangular
-    Optional_int_const MapNum,
-    // Optional< Real64 > MapWindowSolidAngAtRefPt, //Inactive
-    Optional<Real64> MapWindowSolidAngAtRefPtWtd)
+    int const MapNum)
```

Comment:
Get rid of all `optional` arguments - either remove if unused or change type and add a default if need be.